src/app/blog/multipath-will-fix-this/page.mdx

```diff
@@ -0,0 +1,308 @@
+import { BlogPostLayout } from '@/components/BlogPostLayout'
+import {ThemeImage} from '@/components/ThemeImage'
+
+export const post = {
+  draft: false,
+  author: 'Floris Bruynooghe',
+  date: '2025-07-29',
+  title: 'Multipath Will Fix This',
```

Comment:
So this title is basically an in-joke of the team.  Which isn't even explained in the post.  While that's a great working title, and is probably fine as a URL slug, I'm not sure this works as an actual title.